src/styles/components/card.scss

```diff
@@ -0,0 +1,70 @@
+.card {
+  box-sizing: border-box;
+  @include flex-column;
+  padding: 32px 16px 16px;
+  max-width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: $white-color;
+
+  &__content {
+    @include flex-column;
+    margin-top: 40px;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__description {
+    color: $secondary-color;
+    margin: 4px 0 16px;
+  }
+
+  &__stars-container {
+    @include flex-center-between;
+  }
+
+  &__price {
+    @include flex-center-between;
+    margin: 24px 0 16px;
+    &-text {
+      color: $secondary-color;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-num {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__btn {
+    font-size: 14px;
+    font-weight: 700;
+    text-transform: uppercase;
+    padding: 12px 0;
+    text-align: center;
+    background-color: $hover-color;
+    color: $white-color;
+    border-radius: 5px;
+    text-decoration: none;
+    line-height: 16px;
+    &:hover {
+      background-color: $white-color;
+      color: $hover-color;
+      border: 1px solid $hover-color;
```

Comment:
9'th point in [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)